modules/bmc/ssh.rb

```diff
@@ -60,6 +60,36 @@ def powerstatus
       def powercycle
         ssh(@powercycle)
       end
+
+      def ip
+        host
+      end
+
+      # the following are dummy implementations
+
+      def mac
+        ''
```

Comment:
Perhaps instead of empty strings appropriate dummy values would work better? Something like ```00:00:00:00:00:00``` for mac address, ```127.0.0.1``` for gateway, etc?